.github/SUPPORT.md

```diff
@@ -3,27 +3,40 @@
 Thanks for using `{report}`. Before filing an issue, there are a few places
 to explore and pieces to put together to make the process as smooth as possible.
 
-Start by making a minimal **repr**oducible **ex**ample using the
-[reprex](http://reprex.tidyverse.org/) package. If you haven't heard of or used
-reprex before, you're in for a treat! Seriously, reprex will make all of your
-R-question-asking endeavors easier (which is a pretty insane ROI for the five to
-ten minutes it'll take you to learn what it's all about). For additional reprex
-pointers, check out the [Get help!](https://www.tidyverse.org/help/) resource
+1. Try to see if updating to the latest versions of the *easystats* packages help solve 
+your issue. You can do so with: `easystats::install_latest()`.
```

Comment:
Maybe also mention that they shouldn't forget to refresh the R session before trying again?